backend/pom.xml

```diff
@@ -259,6 +259,54 @@
                     <suspend>false</suspend>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>com.github.eirslett</groupId>
+                <artifactId>frontend-maven-plugin</artifactId>
+                <version>1.11.2</version>
+                <configuration>
+                    <workingDirectory>${project.basedir}/src/main/webapp</workingDirectory>
+                    <installDirectory>target</installDirectory>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>install node and npm</id>
+                        <goals>
+                            <goal>install-node-and-yarn</goal>
+                        </goals>
+                        <configuration>
+                            <nodeVersion>v14.17.1</nodeVersion>
+                            <yarnVersion>v1.22.10</yarnVersion>
```

Comment:
Can't we use a more recent yarn version? This one is pretty old.

Reply:
I read that there is some migration effort into moving to yarn 2.0.0 (but didn't really look into the details).
I'll try to update to the last minor version and take some time to migrate across all our frontend apps to 2.0 if it makes sense and not too difficult.

Reply:
1.22.10 is actually the latest stable version. The `berry`  (aka 2.0) is a bit older and the last published tag is `rc`.
I'm not really sure if it's stable or not :)